src/olympia/blocklist/admin.py

```diff
@@ -135,7 +135,8 @@ class BlocklistSubmissionAdmin(admin.ModelAdmin):
     ordering = ['-created']
     view_on_site = False
     list_select_related = ('updated_by', 'signoff_by')
-    change_form_template = 'blocklist/blocklistsubmission_change_form.html'
+    change_form_template = \
+        'admin/blocklist/blocklistsubmission_change_form.html'
```

Comment:
we don't generally use this coding style in this project - we tend to wrap in `(` `)` where needed instead.
```suggestion
    change_form_template = (
        'admin/blocklist/blocklistsubmission_change_form.html')
```